Index: src/LAB2/BasicJave.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LAB2/BasicJave.java	(date 1608004068834)
+++ src/LAB2/BasicJave.java	(date 1608004068834)
@@ -0,0 +1,21 @@
+package LAB2;
+
+public class BasicJave {
+
+    public static void main(String[] args) {
+        System.out.println("Data Type in Programming.");
+        //Integer จำนวนเต็ม เช่น -100 -10 -2 -1 0 1 2 3 10 100
+        int number;
+        number = 100; // ตัวแปร number เก็บค่าข้อมูลจำนวนเต็ม 100
+
+        System.out.println(number);
+
+        number = number *100;
+        System.out.println(number);
+
+
+    }//main
+
+
+
+}//calss
