Index: src/LAB2/Exercise_STD_Profile.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LAB2/Exercise_STD_Profile.java	(date 1609216207865)
+++ src/LAB2/Exercise_STD_Profile.java	(date 1609216207865)
@@ -0,0 +1,25 @@
+package LAB2;
+
+import java.util.Scanner;
+
+public class Exercise_STD_Profile {
+
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter message: ");
+        int x =Integer.parseInt(sc.nextLine());
+
+        System.out.println("sasina Nimkhon");
+        System.out.println("362211760025");
+        System.out.println("22");
+        System.out.println("0980945964");
+        System.out.println("mindnn2541@gmail.com");
+
+
+
+
+
+
+    }//main
+}//class
Index: src/LAB2/BasicDataType.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LAB2/BasicDataType.java	(date 1609213889237)
+++ src/LAB2/BasicDataType.java	(date 1609213889237)
@@ -0,0 +1,50 @@
+package LAB2;
+
+public class BasicDataType {
+
+    public static void main (String[] args){
+
+        // boolean => T,F (1,0)
+        boolean myboo = true;
+        System.out.println(myboo);
+        myboo = false;
+        System.out.println(myboo);
+
+        //character =>only 1 character such as 'a', 'x', '$'
+        char c ='a';
+        System.out.println(c);
+
+        //integer => -2 -1 0 1 2 3 ... 10000
+        int number;
+        number = 100;
+        System.out.println(number); //100
+        System.out.println(number*5); //500
+        //number = ? 100
+
+        //real number => float, double => 10.11, -22.1
+        float a = 10.10f;
+        double b = 20.20;
+        System.out.println(a+b); //30.30
+
+        //String => message => "...." => "Hello"
+        //String คือ ข้อมูลชนิดแบบข้อความ ที่มีตัวอักษรต่อเรียงกัน
+        String msg = "Hello RUTS Saiyai 2020";
+        System.out.println(msg);
+        System.out.println(msg.toUpperCase());
+        System.out.println(msg.length());
+
+        //constant variable
+
+        final int num = 100;
+
+
+
+
+
+
+
+
+
+
+    }//main
+}//class
Index: src/LAB2/BasiclnputData.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LAB2/BasiclnputData.java	(date 1609215238867)
+++ src/LAB2/BasiclnputData.java	(date 1609215238867)
@@ -0,0 +1,25 @@
+package LAB2;
+
+import java.util.Scanner;
+
+public class BasiclnputData {
+
+    public static void main(String[] args){
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter an integer: ");
+        int num = scanner.nextInt();
+
+        System.out.println("The number is "+num);
+
+
+
+
+
+
+
+
+
+    }//main
+}//calls
Index: src/LAB2/BasicJava.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/LAB2/BasicJava.java	(date 1609211073000)
+++ src/LAB2/BasicJava.java	(date 1609211365194)
